CONTRIBUTING.md

```diff
@@ -28,6 +29,7 @@ If you would like to contribute to this repository by adding features, enhanceme
     - [Build Dependencies](#build-dependencies)
     - [Building the SDK](#building-the-sdk)
     - [Running Tests](#running-tests)
+      - [Running Samples Locally](#running-samples-locally)
```

Comment:
Can we combine this with the section below called "Local Samples Tests"?

Reply:
I kept it separate since that's about running `samples:test` and going into webdriver and this is about `samples:serve`